go.mod

```diff
@@ -2,6 +2,8 @@ module github.com/opendatahub-io/odh-model-controller
 
 go 1.21
 
+toolchain go1.22.4
```

Comment:
Looks like you also need to remove this line. The container images are not building because of this.